### PR TITLE
Translation Spanish  corrected 1102 Words, ch20-06

### PR DESCRIPTION
**Pull request by Santu41**
this is a new Folk with my contribution (1102 Words) and errors solved. 
I hope it is correct! :+1:

### CODE REVIEW

File: second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-they’re working on. Then we’ll implement a way for the `ThreadPool` to tell the
-threads they should stop accepting new requests and shut down. To see this code
-in action, we’ll modify our server to only accept two requests before
-gracefully shutting down its thread pool.
-
-Let’s start with implementing `Drop` for our thread pool. When the pool is
-dropped, we should join on all of our threads to make sure they finish their
-work. Listing 20-22 shows a first attempt at a `Drop` implementation; this code
-won’t quite work yet:
+## Agraciado cierre y limpieza
```
Comment (thecodix) [inline second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md]: Cierre elegante

File: second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+
+El código del listado 20-21 responde a las solicitudes de forma asincrónica
```
Comment (thecodix) [inline second-edition/src/ch20-06-graceful-shutdown-and-cleanup.md]: asincrona

**Comment by thecodix** [comment]
Many thanks and very sorry for the extra hassle!

I left very minor comments, please review and update, will merge after it :)

Keep up the good work!

**Comment by Santu41** [comment]
Done!

**Comment by thecodix** [comment]
Please keep working on it :)

**Comment by Santu41** [comment]
Hello Alfredo, I do not know where to contact you .. I wanted if you can let me know if the translations I'm doing right now are correct because I will not be able to connect again until tomorrow morning and I want to leave them in the best possible conditions!

**Comment by thecodix** [comment]
Reach me at romeu.alfred@gmail.com

**Comment by thecodix** [comment]
machine translated

**Pull request merged**